### DISCUSSION TITLE
Page reloading for no reason when route switching

### DISCUSSION THREAD

**Discussion by eevaery** [nuxt/nuxt#25424] [Questions]
When switching routes it may cause the page to reload, reflected in a white screen, which lasts for about 2-5s, I'd like to ask if any of my partners have encountered the same problem as me or ask for assistance

The total number of all page files: 61

**Comment by eevaery** [comment]
Just to add, my ssr is off.

**Comment by AloisSeckar** [comment]
For example you can have a global route middleware set up that is waiting for something? Or are you sure, that you are not doing full reload each time? Hard to tell without seeing any code...

Opening dev console, going to Network tab and observe, what is happening when you navigate along your application, might give you some clues.

**Reply by eevaery** [reply]
In the console you can see that all the output is reset, I'm sure it happened with a full reload, the code part may not be very convenient, I currently have an authentication role of the middleware that exists
